code/game/objects/effects/landmarks/landmarks.dm

```diff
@@ -183,6 +183,18 @@
 	. = ..()
 	return INITIALIZE_HINT_QDEL
 
+/obj/effect/landmark/zone_to_control
+	name = "Zone to control landmark"
+	icon_state = "holding_facility"
+
+/obj/effect/landmark/zone_to_control/Initialize()
+	var/area/area_to_control = get_area(src)
+	area_to_control.set_to_contested()
+	area_to_control.center_turf = get_turf(src)
+	GLOB.zones_to_control += area_to_control
+	. = ..()
```

Comment:
```suggestion
	..()
```


Or alternatively why not just attach the overlay to this obj and leave it

Reply:
hum, yeah ok